test/BUILD.gn

```diff
@@ -146,6 +147,7 @@ test("brave_unit_tests") {
   ]
 
   defines = brave_service_key_defines
+  defines += brave_updater_defines
```

Comment:
you shouldn't do this. `brave_updater_defines` and `brave_service_key_defines` should either go in a public config of the component updater target or go in `//brave/build:compiler`

Reply:
resolved!